package.json

```diff
@@ -25,7 +25,7 @@
   "bugs": "https://github.com/aldy505/vite-plugin-pages-solid/issues",
   "scripts": {
     "dev": "npm run build -- --watch --ignore-watch examples",
-    "build": "npm run build-cjs && npm run build-esm",
+    "build": "concurrently --kill-others-on-fail \"npm:build-*\"",
```

Comment:
thanks, but sorry I don't intend to add anymore dependency.

making the build time fast is not my goal, we can wait it to build, because it's fast already, it doesn't take any longer than 10 seconds to do so.

Reply:
This is a only dev dependency